src/fragments/lib/storage/ios/upload.mdx

```diff
@@ -58,7 +58,7 @@ let resultSink = uploadTask
 ## Upload file
 
 When you have a file that you want to upload, you can specify the url to the file in the `local` parameter.
-
+If a file with the same `filenameKey` already exists, the existing file will be overwritten.
```

Comment:
```suggestion
If a file with the same `key` already exists, the existing file will be overwritten.
```
wdyt about use `key` instead? `filenamekey` is only mentioned in the code example and IIUC this is referring to files in the S3 bucket being overwritten.